src/components/application_manager/test/commands/mobile/set_global_properties_request_test.cc

```diff
@@ -575,14 +643,134 @@ TEST_F(SetGlobalPropertiesRequestTest, Run_TTSIncorrectSyntax_Canceled) {
   command->Run();
 }
 
+TEST_F(SetGlobalPropertiesRequestTest, Run_InvalidHelpPromptText_INVALID_DATA) {
+  MessageSharedPtr msg = CreateMsgParams();
+  SmartObject help_prompt(smart_objects::SmartType_Array);
+  help_prompt[0][strings::text] = "help_prompt\\n";
+  (*msg)[strings::msg_params][strings::help_prompt] = help_prompt;
+
+  ExpectVerifyImageVrHelpUnsuccess();
+  EmptyExpectationsSetupHelper();
+
+  SharedPtr<SetGlobalPropertiesRequest> command(
+      CreateCommand<SetGlobalPropertiesRequest>(msg));
+
+  ExpectInvalidData();
+
+  command->Run();
+}
+
+TEST_F(SetGlobalPropertiesRequestTest, Run_InvalidVrHelpText_INVALID_DATA) {
+  MessageSharedPtr msg = CreateMsgParams();
+  SmartObject vr_help(smart_objects::SmartType_Array);
+  vr_help[0][strings::text] = "vr_help\\n";
+  (*msg)[strings::msg_params][strings::vr_help] = vr_help;
+
+  ExpectVerifyImageVrHelpSuccess(vr_help);
+  EmptyExpectationsSetupHelper();
+
+  SharedPtr<SetGlobalPropertiesRequest> command(
+      CreateCommand<SetGlobalPropertiesRequest>(msg));
+
+  ExpectInvalidData();
+
+  command->Run();
+}
+
+TEST_F(SetGlobalPropertiesRequestTest, Run_InvalidImageValue_INVALID_DATA) {
+  MessageSharedPtr msg = CreateMsgParams();
+  SmartObject vr_help(smart_objects::SmartType_Array);
+  vr_help[0][strings::text] = "vr_help";
+  vr_help[0][strings::image][strings::value] = "value\\n";
```

Comment:
@okozlovlux Would be nice to specify in values why they are not correct:
Example:
"value with empty line in the end \\n"